### PR TITLE
Fix compilation and sync utests results

### PR DESCRIPTION
**Pull request by IvanNardi**

### CODE REVIEW

File: src/include/ndpi_typedefs.h [hunk 1]
```diff
@@ -1572,8 +1572,8 @@ struct ndpi_flow_struct {
 _Static_assert(sizeof(((struct ndpi_flow_struct *)0)->protos) <= 200,
                "Size of the struct member protocols increased to more than 200 bytes, "
                "please check if this change is necessary.");
-_Static_assert(sizeof(struct ndpi_flow_struct) <= 904,
-               "Size of the flow struct increased to more than 904 bytes, "
+_Static_assert(sizeof(struct ndpi_flow_struct) <= 920,
```
Comment (utoni) [inline src/include/ndpi_typedefs.h:1575]: 916 should be enough, theoretically...

Comment (utoni) [reply src/include/ndpi_typedefs.h:1575]: forgot about 8 byte padded structs..

**Review by utoni — APPROVED** [review]

File: src/include/ndpi_typedefs.h (hunk 1 above)
Comment (utoni) [reply src/include/ndpi_typedefs.h:1575]: I think I've got it, structs are still 4-byte padded, but you thought ahead due to #1761.

Comment (IvanNardi) [reply src/include/ndpi_typedefs.h:1575]: Not thinking ahead, sorry :)
I simply checked:
```
ivan@ivan-Latitude-E6540:~/svnrepos/nDPI(utets)$ ../pahole/pahole -a src/lib/libndpi.so | grep ndpi_flow_struct -A 460
[...]

	u_int8_t                   ovpn_session_id[8];   /*   878     8 */
	u_int8_t                   ovpn_counter;         /*   886     1 */
	u_int8_t                   tinc_state;           /*   887     1 */
	u_int16_t                  flow_payload_len;     /*   888     2 */

	/* XXX 6 bytes hole, try to pack */

	/* --- cacheline 14 boundary (896 bytes) --- */
	char *                     flow_payload;         /*   896     8 */
	u_int8_t                   priv_data[16];        /*   904    16 */

	/* size: 920, cachelines: 15, members: 77 */
	/* sum members: 891, holes: 5, sum holes: 22 */
	/* sum bitfield members: 53 bits, bit holes: 2, sum bit holes: 3 bits */
	/* paddings: 1, sum paddings: 4 */
	/* last cacheline: 24 bytes */

**Pull request merged**